docs/getting_started/index.rst

```diff
@@ -1,32 +1,238 @@
+           :language: bash
+
+        When doing so, it will print output equivalent to the following.
+
+        .. literalinclude:: build_example_linux_spdk.out
+           :language: yaml
+
+    .. tab:: FreeBSD
+
+        .. literalinclude:: build_example_freebsd.cmd
+           :language: bash
+
+        When doing so, it will print output equivalent to the following.
+
+        .. literalinclude:: build_example_freebsd.out
+           :language: yaml
+
+    .. tab:: macOS
+
+        .. literalinclude:: build_example_macos.cmd
+           :language: bash
+
+        When doing so, it will print output equivalent to the following.
+
+        .. literalinclude:: build_example_macos.out
+           :language: yaml
+
+    .. tab:: Windows
+
+        .. literalinclude:: build_example_windows.cmd
+           :language: powershell
+
+        When doing so, it will print output equivalent to the following.
+
+        .. literalinclude:: build_example_windows.out
+           :language: yaml
+
+        .. note:: Run this in an elevated Powershell session!
+
+As the compilation and execution of the :ref:`sec-gs-example` demonstrate,
+platform and interface-specific concerns exist. However, the **xNVMe** API
+and library encapsulate these differences, enabling the same example code to
+function uniformly across all platforms. This provides a unified platform and
+system interface abstraction.
+
+The library encapsulation in **xNVMe** is referred to as :ref:`sec-backends`.
+In the example above, only a fraction of the supported system interfaces are
+shown. The section on :ref:`sec-backends` provide additional documentation on
+this, as well as describing other runtime concerns such as system setup and
+configuration.
+
+With the library and toolchain in place, you might want to explore
+the :ref:`sec-tools` and :ref:`sec-api`. For this, you can use your own storage
+devices. However, you can also look at the :ref:`sec-tutorials` section,
+specifically the :ref:`sec-tutorials-devs-linux` section, which will enable you
+to develop and test using emulated storage devices with support for features
+such as such as Zoned Namespaces, Key-Value, and Flexible-Data-Placement.
+
+In case things aren't working for you then have a look at the following section
+on :ref:`sec-gs-troubleshooting`.
+
+.. _sec-gs-troubleshooting:
+
+Troubleshooting
+===============
+
+This section is here to give you pointers to troubleshoot whether **xNVMe** or
+your system is misbehaving. First off ensure that you are you have the setup
+your system, please refer to the :ref:`sec-toolchain` section for details
+on installing compilers, libraries etc. and the :ref:`sec-backends` section
+on notes on system configuration and runtime instrumentation of the library
+backends.
+
+Should you not find the answers you are looking here, then please
+reach out by raising an :xref-github-xnvme-issues:`issue<>`, start
+an asynchronous :xref-github-xnvme-discussions:`discussions<>`, or go
```

Comment:
I think we should consider dropping this suggestion. In the past we have been really bad at reacting to discussions on Github. Perhaps it is easier to do everything either as Github issues or on Discord

Reply:
sure.